server/eclipse-project/src/main/java/us/freeandfair/corla/csv/DominionCVRExportParser.java

```diff
@@ -223,6 +235,20 @@ private void addContests(final CSVRecord the_choice_line,
   }
   
   /**
+   * Checks to see if the set of parsed CVRs needs flushing, and does so 
+   * if necessary.
+   */
+  private void checkForFlush() {
+    if (my_parsed_cvrs.size() % BATCH_SIZE == 0) {
```

Comment:
See comment above about how this is being checked.

More generally, can `BATCH_SIZE` and some of this parsing logic be factored out?